features/apimgt/org.wso2.carbon.apimgt.store.feature/src/main/resources/admin/site/themes/wso2/templates/alert-table/js/alert-table.js

```diff
@@ -46,14 +46,12 @@ $( document ).ready(function() {
         } else if(selectedOptionKey== 4){
             tableName = "ORG_WSO2_ANALYTICS_APIM_ABNORMALRESPONSETIMEALERTSTREAM";
```

Comment:
Are these table names correct? I checked the STATS_DB tables and it does not have ORG_WSO2_ANALYTICS_APIM prefix

Reply:
Actually, these tables were old one. all the mappings to the latest tables are done in the /modules/stats/stats.jag 

https://github.com/wso2/carbon-apimgt/blob/6.x/features/apimgt/org.wso2.carbon.apimgt.store.feature/src/main/resources/admin/modules/stats/stats.jag#L218

Reply:
Better to remove old table names. will update the PR with repalced one

Reply:
Yes. That would be better

Reply:
updated the table names